Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py

```diff
@@ -79,9 +110,12 @@ def convert_date_to_timestamp(date):
     if TIME_METHOD == 'Timestamp-Seconds':
         return int(date.timestamp())
 
-    else:  # this case includes "Timestamp-Milliseconds"
+    elif TIME_METHOD == 'Timestamp-Milliseconds':
```

Comment:
You returned in the if block above, so this can simply be an `if` (and not an `elif`.)